projects/managed-hubs/support.md

```diff
@@ -173,24 +174,87 @@ This process is carried out in an ongoing basis by the {term}`Support Stewards`.
+
+2. `Who: Support steward`
+
+   **Spend 30 minutes trying to resolve**. If you believe you can resolve the issue within 30 minutes, try resolving it yourself.
+   1. If you resolve the issue, then jump to the "Confirm resolution" step 7.
+   2. If you don't believe you can resolve the issue (or you couldn't) in 30 minutes, jump to the next step.
+
+3. `Who: Support Steward`
+
+   **Open an engineering issue**. If this is a {term}`Change Request` or {term}`Guidance Request` and/or you cannot resolve the issue within 30 minutes, then open a support issue for the team to discuss.
+
+   [{bdg-primary}`Open a general issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=&template=01_new-issue.yml) that describes the request and any next steps we should take. It should be automatically added to the **Eng & Prod** board by the existing automation. Add the **type**: `support` and choose an **impact** level in the **Eng & Prod** board metadata (available for the very same issue, no need to visit the board itself!).
+   
+   If the issue has a `critical` impact (we defer that first evaluation to the support steward), an additional ping to the support Slack channel is needed to boost the signal.
+   
+   ```{admonition} What does `critical` mean?
+
+   We recognize there might be some support-related issues that do not count as [incidents](incidents:what), but 
+   they need a quick resolution (inside the current sprint window) because they are impacting the execution of 
+   desired or existing workflows (degraded experience) for our communities.
+   Examples of those sorts of issues (requests) are:
+     * Image refs updates
+     * Profile updates
+     * User storage limitations
+     * Grafana (and Prometheus) failures
+   Additionally and depending on the nature AND context of the issue (request):
+     * Access to specific buckets
+     * Authentication and authorization updates
+   
+   The support steward **should** self-assign the `critical` issue and work on it immediately (this is now outside of the 30-minute timebox described in step 2).
+   
+   If the support stewards (both of them) do not have the capacity to resolve the `critical` issue (ie. working on another `critical` issue, being out of their working time, etc.), they should ping the **Engineering Manager** (or the delegated person) so they can secure resources to resolve that issue on the fly (see step 7 below).
+   
+   The support steward **should not** work on issues with impact lower than `critical` (unless they are assigned as part of the "planned" reactive work in the context of a running sprint (see step 6 below).
+
+4. `Who: Partnerships representative and the Engineering Manager (or respective delegates)`
+
+   **Revisit the impact metadata**. Once a week (at minimum) the [support view in the **Eng & Prod** board](https://github.com/orgs/2i2c-org/projects/22/views/47) should be revisited to validate the impact level on support-related issues.
```

Comment:
```suggestion
   **Revisit the impact metadata**. Once a week (at minimum) the [support view in the **Eng & Prod** board](https://github.com/orgs/2i2c-org/projects/22/views/47) should be revisited to validate the impact level on support-related issues. Currently, we allocate a 30-minute working session every Wednesday (open to everyone to participate) to perform such prioritization.
```

Reply:
More specificity on the revision working session as per suggestion from @yuvipanda.